Audiobook Player/PlayerViewController.swift

```diff
@@ -202,35 +208,28 @@ class PlayerViewController: UIViewController {
     }
 
     @IBAction func didSelectChapter(_ segue: UIStoryboardSegue) {
-
         guard PlayerManager.sharedInstance.isLoaded() else {
```

Comment:
let's use this guard to unwrap the optionals for the viewController and currentChapter